docs/source/examples/plot_forecasting.py

```diff
@@ -285,18 +286,20 @@
 ### We now run IPF for the future vectors
 
 # %%
-args = {'matrix': demand,
-        'rows': vectors,
-        'columns': vectors,
-        'column_field': "destinations",
-        'row_field': "origins",
-        'nan_as_zero': True}
+args = {
+    "matrix": demand,
+    "rows": vectors,
+    "columns": vectors,
+    "column_field": "destinations",
+    "row_field": "origins",
+    "nan_as_zero": True,
+}
 
 ipf = Ipf(**args)
 ipf.fit()
 
-ipf.save_to_project(name='demand_ipf', file_name='demand_ipf.aem')
-ipf.save_to_project(name='demand_ipf_omx', file_name='demand_ipf.omx')
+ipf.save_to_project(name="demand_ipfd", file_name="demand_ipfd.aem")
```

Comment:
why are these types of changes mixed in?

Reply:
Black.  My auto-commit hook would not allow me to do the commit without it.  And I tried!